app/styles/_navbar-vertical.less

```diff
@@ -87,6 +87,12 @@
       @media(max-width: @screen-xs-max) {
         padding: 10px 15px 10px 5px;
       }
+      .badge {
```

Comment:
I had absolutely positioned .badge in one of my PRs.  It looks like it got rebased away.  :(  Can we redo the absolute positioning?  Otherwise the width of the nav item changes with the coming and going of the badge..m

Reply:
Nevermind.  The changes are there.  But perhaps you should combine the two rules so they're not in two different files?